services/shhext/service_test.go

```diff
@@ -72,6 +73,8 @@ type ShhExtSuite struct {
 }
 
 func (s *ShhExtSuite) SetupTest() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Wouldn't it be better to use the same seed for every test run?

Reply:
The only thing I was thinking is that if we start publishing on a public channel, we want the channel to be different at each test run, otherwise we'd get clashes

Reply:
@adambabik below in the test, we use it to create a random string , which we treat as a publicChat which we then join

Reply:
> The only thing I was thinking is that if we start publishing on a public channel, we want the channel to be different at each test run, otherwise we'd get clashes

The better strategy would be to send messages, remember their hashes and request messages back checking if the expected hashes got received.

Also, the messages really shouldn't leave the local node so I don't see a reason to clash with anything.

If we can avoid using `rand` in tests, let's do it because it might lead to issues with debugging and reproducing failures as well as increases a chance to get flaky tests.